boards/arduino/nicla_vision/arduino_nicla_vision.dtsi

```diff
@@ -31,3 +31,22 @@
 &mailbox {
 	status = "okay";
 };
+
+sdhc: &sdmmc2 {
+	compatible = "st,stm32-sdio";
+	interrupts = <124 0>;
+	interrupt-names = "event";
+	pinctrl-0 = <&sdmmc2_d0_pb14 &sdmmc2_d1_pb15
+			&sdmmc2_d2_pg11 &sdmmc2_d3_pb4
+			&sdmmc2_ck_pd6 &sdmmc2_cmd_pd7>;
+	pinctrl-names = "default";
+	min-bus-freq = <DT_FREQ_K(400)>;
+	max-bus-freq = <DT_FREQ_M(208)>;
+	status= "okay";
+	wifi: airoc-wifi {
```

Comment:
Nitpicking: better with a empty line above. Ditto in **arduino_portenta_h7-common.dtsi**.